src/internal/util/lift.ts

```diff
@@ -17,16 +9,5 @@ export function hasLift(source: any): source is { lift: InstanceType<typeof Obse
 export function operate<T, R>(
   init: (liftedSource: Observable<T>, subscriber: Subscriber<R>) => (() => void) | void
 ): OperatorFunction<T, R> {
```

Comment:
```suggestion
export function operate<T, R>(
  init: (source: Observable<T>, subscriber: Subscriber<R>) => (() => void) | void
): OperatorFunction<T, R> {
```

Probably JSDoc should also be updated.

Reply:
I'll address this in a followup.